app/src/main/AndroidManifest.xml

```diff
@@ -1,12 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:installLocation="auto"
     package="org.kiwix.kiwixmobile">
 
+  <supports-screens
```

Comment:
get rid of both of these. I assume an interactive rebase went weird on you?

Reply:
Fixed e792391583725bc1aadc3e6ff5a04d04b6985ce9